auth_otp/services/kavenegar_service.py

```
@@ -0,0 +1,232 @@
+from django.conf import settings
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class KavenegarService:
+    """
+    سرویس ارسال پیامک با کاوه‌نگار
+    """
+    
+    def __init__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)